### ISSUE TITLE
Develop page for 404 Error Invalid URL page

### ISSUE THREAD

**Issue by qaz6209031** [hackforla/website#1482]
### Overview 
It's a good idea to create a page for invalid url, so instead of seeing 404, user could have more clue to see what's going on. 

### 
<details>
<summary>Details </summary>

![Screen Shot 2021-04-27 at 8 09 45 PM](https://user-images.githubusercontent.com/44656685/116340764-9151fd00-a794-11eb-8195-120a0f873ee4.png)
</details>

RIght now if the invalid url is provided, out page will look like this 

<details>
<summary>We could improve it to something like youtube did:</summary>

![Screen Shot 2021-04-27 at 8 11 31 PM](https://user-images.githubusercontent.com/44656685/116340942-e0982d80-a794-11eb-81e0-8f9f90d2b3ae.png)
</details>

or 
<details>
<summary>Netflix</summary>

![Screen Shot 2021-04-27 at 8 12 49 PM](https://user-images.githubusercontent.com/44656685/116340997-f60d5780-a794-11eb-8bb1-e99360900d96.png)
</details>

### Action Items  

 Design
- [x] Finalise the web version
- [x] Finalise the mobile version 

Development 
- [x] Develop the new 404 Error page for the HfLA site matching final Figma design specifications (in the pink rectangle https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=9616%3A0)
- [x]  Add illustration credit info (credit details listed below Figma design) to [Credits page ](https://www.hackforla.org/credits/)

### Resources  
[404 Error page in Figma (final design in pink rectangle)](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=9616%3A0)
[Template of a credit.yml file](https://github.com/hackforla/website/wiki/Template-of-a-credit.yml-file-(for-Credits-page))
[Credits page](https://www.hackforla.org/credits/)
Sample invalid url (Hack for LA): https://www.hackforla.org/hahaha
Sample invalid url (Youtube): https://www.youtube.com/asd
Sample invalid url (Netflix): https://www.netflix.com/hahaha

**Comment by akibrhast** [comment]
@qaz6209031 This is a great idea!
@daniellex0 Can the design team cook up something?

**Comment by daniellex0** [comment]
Sure, I'll add UI/UX labels and see if anyone wants to take this on! Good idea @qaz6209031 ! 👍

**Comment by qiqicodes** [comment]
@Rosalyn-broddie Can you let us know your progress in regards to this issue? Thank you =]

Progress:
Blockers:

**Comment by Rosalyn-broddie** [comment]
Progress: Started looking for more examples to pull inspiration from
Blocks: Haven't gotten a chance to start designing the page yet
Availability: I'll have some extra time this week to start designing some options for this page
ETA: Uncertain

**Comment by Rosalyn-broddie** [comment]
Progress: Designed a couple of potential options for the 404 error page, but haven't fleshed them out yet
Blocks: Stopped working on the 404 page for a little while in order to focus attention on the homepage
Availability: I'll have some time to work on some more options this week
ETA: 1 week

**Comment by Rosalyn-broddie** [comment]
Progress: Have a couple of options ready for an MVP, but I'll continue working on other options (animated, gifs, etc.) for the future
- Option 1: https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?page-id=9616%3A0&node-id=9630%3A97224&viewport=1648%2C761%2C0.284930020570755&scaling=min-zoom
- Option 2: https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?page-id=9616%3A0&node-id=10259%3A95726&viewport=1648%2C761%2C0.284930020570755&scaling=min-zoom

Availability: Since the focus has been on the homepage lately, I may not have extra time this week for other options, but I'll do some work on it if I do
ETA: MVP options are finished, but I'll continue to look for more options that could be more fun (gifs, etc.)

**Comment by Sihemgourou** [comment]
@Rosalyn-broddie 
Please add update
1. Progress
2. Blockers
3. Availability
4. ETA

**Comment by Sihemgourou** [comment]
This looks good, great job @Rosalyn-broddie we can move this to development ! 

Lets go with this option : https://www.figma.com/proto/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?page-id=9616%3A0&node-id=9630%3A97224&viewport=1648%2C761%2C0.284930020570755&scaling=min-zoom

**Comment by akibrhast** [comment]
@Rosalyn-broddie Could you please restructure the Figma designs for the error page such that it uses the grid system. Using the grid system is typically more industry standard and on top of that it also allows for easier creation of responsive UI, faster.

- Please componentize the Figma Designs as well. Such that the spaces between the two separate components are clear and distinct.
- The page itself and the component hierarchy on the left also needs to be cleaned up.
- Please provide the image as a separate component, that is easily clickable and exportable. Please look at the below image. As a developer, I am not sure which is the right image that should be exported. Should I export `freepic background comp` should I export `no data-pana 2` . On top of that, what is `no data-pana 2` how am I supposed to understand from the name of that asset that, that is correct asset needed for this page ?
![image](https://user-images.githubusercontent.com/32349001/121946187-ac67d480-cd22-11eb-8aab-816c2879d2c0.png)
- As a developer it is also unknown to me what format should I export the asset in, should it svg,png, jpeg, gif ? Could this please be specified in the figma file

Something like this:
- https://www.figma.com/file/zk9cwbZqGC24IxLTCLQ1Lu/Hack-for-LA-Website-Copy?node-id=9616%3A0

**Comment by sena-ji** [comment]
1. Progress: Thought it would be a good challenge for me to work on this issue. Asked Akib for some points to help me get started. Will talk about it on the meeting on Tuesday.
2. Noted above.
3. Available to work on it this week.
4. Will get a pull request up by next week's meeting for review.

**Comment by sena-ji** [comment]
1. Progress: I added the image used to the credits page. I created a skeleton of the page seen below:
![image](https://user-images.githubusercontent.com/42919318/124510691-37b81100-dd89-11eb-8822-0fa501f10189.png)
2. Blockers: I think the fonts and sizes used on Figma are not showing correctly on my screen. Also need some help with container sizing. 
3. Availability: Will be asking for some help on Slack and the team dev meeting if someone is available.
4. ETA: Hopefully by dev team meeting this week!

**Comment by averdin2** [comment]
@sena-ji I spoke to Danielle about the font issue. She said not to worry about it for now. I think that there is a larger issue regarding the font that I am going to try and look into.

**Comment by sena-ji** [comment]
@averdin2 Thank you for the quick update! I will disregard the font for now and work on the mobile view :)

**Comment by sena-ji** [comment]
1. Progress: I made some adjustments to the mobile view to align with Figma.

2. Blockers: I did some research to try and rearrange items on mobile view. Having a hard time getting the image to align between text boxes, Asked Alex for some help.

3, Availability: Will work on the issue throughout the week! Been lagging because of work.

Here is the mobile preview at the moment:
![image](https://user-images.githubusercontent.com/42919318/125499397-1b6dbbe8-a5ce-4e84-a229-1d7dffcc8251.png)
